### DISCUSSION TITLE
Attribute error

### DISCUSSION THREAD

**Discussion by SarveshVGharat** [scholarly-python-package/scholarly#495] [Q&A] [answered]
Why is scholarly giving an attribute error? I tried search_org, search_pubs, etc and in every case, it gave the same error

----> 8 search_results = scholarly.search_org(search_query)
     10 # Define the CSV file and header
     11 csv_file = "search_results.csv"

AttributeError: module 'scholarly' has no attribute 'search_org'

**Comment by arunkannawadi** [comment, marked as answer]
Did you run
```python
from scholarly import scholarly
```

or 
```python
import scholarly
```

It looks like you might have done the latter but you need to do the former.

**Comment by SarveshVGharat** [comment]
Yea my bad import scholarly was what I did, the other one worked fine
Thanks
